types/eslint-ast-utils/index.d.ts

```diff
@@ -0,0 +1,150 @@
+import * as ESLint from "eslint";
+import * as ESTree from "estree";
+
+declare const astUtils: {
```

Comment:
The upstream package just uses a plain export object; these can just be top level `export function` declarations: https://unpkg.com/browse/eslint-ast-utils@1.1.0/index.js

The only reason to do an `export =` is if the upstream package does `module.exports` on something funky, like a function/class that has been modified with properties or something.

Reply:
Gotcha, updated!